Altalite.py

```diff
@@ -7,35 +7,43 @@
 
 collections.Callable = collections.abc.Callable
 
-print("---------------------------------------------------------------------------------")
-print("Welcome in app Altalite!\n"
-      "Enter below the ticker of the company you are interested in and I will analyze it for you")
-print("---------------------------------------------------------------------------------")
 
-CompanyTicker = input("Enter company ticker:")
+CompanyTicker = input("Enter company ticker you want analyze:")
+
+
+def DisplayTitle():
+    print(Separator())
+    print("Welcome in app Altalite!\n"
+              "Choose program which area of company you are interested in and I will analyze it for you")
+    print(Separator())
+    start()
+
+
+def Separator():
+    print("---------------------------------------------------------------------------------")
 
 
 def start():
     program = [
         "1.BASIC_INFORMATION",
-        "3.RATIO_VALUATION",
```

Comment:
Tutaj pozbyłbym się tych liczb z listy opcji jeżeli jedyne na co wskazują to index. Tym może zajmować się for który masz niżej:
```python
for index, value in program:
  print(index, ". ", value)
```